### PR TITLE
94 add license

### PR DESCRIPTION
**Pull request by rpiw**
Add GNU GLP license. Update README. #94 
Edit: project depends (or will be in a few PR) on 3rd-party libs distributed under GNU GLP. Our license must not cause any conflicts with those.

### CODE REVIEW

**Review by TJB4epam — CHANGES_REQUESTED** [review]
I don't see WHY we have added this license though? Wasn't it agreed to mention this and allow us to retrofit that decision later if we want to?

**Comment by rpiw** [comment]
Added explanation in PR. Mention in readme that decision may be retrofitted in any time.

File: README.md [hunk 1]
```diff
@@ -75,3 +75,8 @@ NOT HERE, NOT IN YOUR LOCAL REPOSITORY.
 1. Every successfully merged PR is worth 10 points.
 1. Every new, passing and meaningful test is worth 1 point.
 1. Every added and meaningful Javadoc is worth 1 point.
+
+
+## License
+Project is distributed under GNU Lesser General Public license.
+License may be changed in a future if all dependencies distributed under GNU LGP license were removed.
```
Comment (TJB4epam) [inline README.md:82]: Please add: Current GNU-licensed dependency is JNativeHook.

Also - bump the POM again. Then this can go in.

**Review by TJB4epam — CHANGES_REQUESTED** [review]
Almost there!

**Pull request merged**